<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ListTracksEditor</class>
 <widget class="QWidget" name="ListTracksEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>773</width>
    <height>78</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel
{
	color: rgb(84, 84, 84);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QFrame" name="frameFormCoverArt">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>60</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayoutFormCoverArt">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelPlay">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
  border-image: url(&quot;:/resource/play-btn20-play-icon@3x.png&quot;);
}

QLabel:hover
{
  border-image: url(&quot;:/resource/play-btn20-play-icon@3x.png&quot;);
}</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="margin">
      <number>16</number>
     </property>
     <property name="indent">
      <number>-1</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTitle">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string>title</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelTime">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>time</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelArtist">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>artist</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelAlbum">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>album</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelGenre">
     <property name="font">
      <font>
       <family>Segoe UI</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>genre</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="labelMore">
     <property name="minimumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>28</width>
       <height>28</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel
{
  border-image: url(&quot;:/resource/play-btn28-popupmenu-n@3x.png&quot;);
}

QLabel:hover
{
  border-image: url(&quot;:/resource/play-btn28-popupmenu-h@3x.png&quot;);
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
